lib/bundler/definition.rb

```diff
@@ -175,7 +175,7 @@ def specs
           raise GemNotFound, "Your bundle is locked to #{locked_gem}, but that version could not " \
                              "be found in any of the sources listed in your Gemfile. If you haven't changed sources, " \
                              "that means the author of #{locked_gem} has removed it. You'll need to update your bundle " \
-                             "to a different version of #{locked_gem} that hasn't been removed in order to install."
+                             "to a different version of #{gem_name} that hasn't been removed in order to install."
```

Comment:
Thank you for making this PR to correct the message. I like how you phrased in PR comment and prefer we change it to that. I prefer the gem version is specified in the call to action to ensure the user knows not to install that version especially if they're quickly skimming the message. So something like:
```
You'll need to update your bundle to a version other than #{locked_gem} that hasn't been removed in order to install.
```
What do you think?

Reply:
ping @alyssais

Reply:
That sounds good to me. I'll update the PR.